backend/src/services/memberAttributeSettingsService.ts

```diff
@@ -66,13 +66,31 @@ export default class MemberAttributeSettingsService {
     return false
   }
 
+  static isMultiSelect(values, options) {
+    // Type must be array
+    if (!Array.isArray(values)) {
+      return false
+    }
+    // If empty array, it is valid
+    if (values.length === 0) {
+      return true
+    }
+    // All values must be in options
+    for (const value of values) {
+      if (!options.includes(value)) {
```

Comment:
Could options be undefined?

Reply:
It should not, but adding a check just in case